From f95662fca27e7e2940c22110335654db46bf9c39 Mon Sep 17 00:00:00 2001
From: John Sullivan <jsgthb@kanargh.org.uk>
Date: Fri, 9 Aug 2019 13:23:15 +0100
Subject: [PATCH] Swap pgc_t's still_time and pg_playback_mode which are
 currently the wrong way round, which prevents some discs from playing.

---
 src/dvdread/ifo_types.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dvdread/ifo_types.h b/src/dvdread/ifo_types.h
index 22ece66..f679d29 100644
--- a/src/dvdread/ifo_types.h
+++ b/src/dvdread/ifo_types.h
@@ -294,8 +294,8 @@ typedef struct {
   uint16_t next_pgc_nr;
   uint16_t prev_pgc_nr;
   uint16_t goup_pgc_nr;
-  uint8_t  still_time;
   uint8_t  pg_playback_mode;
+  uint8_t  still_time;
   uint32_t palette[16]; /* New type struct {zero_1, Y, Cr, Cb} ? */
   uint16_t command_tbl_offset;
   uint16_t program_map_offset;
-- 
2.21.0

